src/Tizen.NUI/src/internal/Interop/Interop.ControlDevel.cs

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2020-2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.NUI
+{
+    internal static partial class Interop
+    {
+        internal static partial class ControlDevel
+        {
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_AccessibilityActivateSignal")]
+            public static extern global::System.IntPtr Dali_Toolkit_DevelControl_AccessibilityActivateSignal(Tizen.NUI.BaseComponents.View.ControlHandle arg1);
```

Comment:
Use the function name of Interop as pascal case.

please refer this PR.
https://github.com/Samsung/TizenFX/pull/2318

Reply:
Done.